packages/components/ng-log-live-tail/src/controller.js

```diff
@@ -27,15 +36,13 @@ export default class LogLiveTailCtrl {
     this.keys = Object.keys(this.logKeys);
     this.setUrlTimeout = null;
 
-    this.logKinds = [];
-    this.LogLiveTail.getLogKinds(this.logKindApiUrl)
-      .then((data) => {
-        this.logKinds = data;
-        this.selectedLogKind = data[0]?.name;
-      })
-      .then(() => this.generateTempUrlSource())
-      .then(() => !this.errorMessage && this.startLog())
-      .catch(this.setErrorMessage);
+    this.$scope.$watch('$ctrl.selectedLogKind', () => {
+      this.clearInterval();
+      this.clearSession();
+      this.generateTempUrlSource()
+        .then(() => !this.errorMessage && this.startLog())
+        .catch(this.setErrorMessage);
+    });
```

Comment:
To avoid using $scope, you could use the $onChanges which will be triggered when the bound value is updated (just beware it is also triggered before the $onInit with an empty value)